payment/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import PaymentView, PaymentTogo, MyPilly
+
+urlpatterns = [
+    path('payment',PaymentView.as_view(), name='payment'),
+    path('payment/togo', PaymentTogo.as_view(), name='payment_togo'),
+    path('mypilly', MyPilly.as_view(), name='mypilly'),
```

Comment:
View를 상속하고 있고 model 클래스와 다르게 View 클래스임을 명확하게 하기 위해서 클래스 이름 끝에 View를 붙여주세요
```suggestion
    path('my-pilly', MyPillyView.as_view(), name='mypilly'),
```

Reply:
붙여서 수정했습니다.